docs/src/main/sphinx/admin/properties-task.rst

```diff
@@ -7,7 +7,7 @@ Task properties
 
 * **Type:** :ref:`prop-type-integer`
 * **Restrictions:** Must be a power of two
-* **Default value:** ``16``
+* **Default value:** Minimum of node physical CPUs and ``32``
```

Comment:
What does this mean?

e.g. if node has 

6 physical CPUs 

6+32 = 38 

I think not .. so what is actually happening

Reply:
I think it chooses whatever is the smaller number .. either the number of physical CPUs or 32 

so 6CPUs -> use 6

16 CPU -> use 6

64 CPUs -> use 32

Right? Use the number of CPUs unless there are more than 32 .. then stick with 32 ..

Reply:
> I think it chooses whatever is the smaller number .. either the number of physical CPUs or 32

Yes. It's `min(32, physical cores)`

Reply:
Maybe

Defaults to ``32`` or the number of physical CPUs of the node, choosing whatever value is smaller.